bin/run-test.sh

```diff
@@ -39,9 +39,14 @@ function run {
 }
 
 # Run test/coverage
-run test
-run test/hooks
-for test in test/standalone/test-*.js ;
+for test in test/*.js ;
```

Comment:
To avoid duplication we should be able to write:
```
for test in test/*.js test/hooks/*.js ;
do
    if [[ ! $(node --version) =~ v0\.12\..* || ! "${test}" =~ .*trace\-koa\.js ]]
    then
        run "${test}"
    fi
done
```